README.md

```diff
@@ -9,5 +9,9 @@
 
 ## How to:
 
+Make scripts are provided to run this application and the required MongoDB using docker. In order for these scripts to start the dev server, you must have a debugger application waiting to attach on port `9229`. This is easily accomplished by running these commands in the VSCode terminal, and updating the `Debugger Auto Attach` setting in VSCode settings to `yes`.
+
 - run: `make debug`
```

Comment:
another helpful command is also `make nuke debug`. Its like a reset (i.e. docker-compose down, remove volumes, etc.) then debug